src/ngraph/runtime/cpu/pass/cpu_fusion.cpp

```diff
@@ -813,7 +813,9 @@ void ngraph::runtime::cpu::pass::CPUFusion::construct_conv_bias_bprop()
                     ngraph::replace_node(m.get_match_root(), goe1);
                     NGRAPH_DEBUG << "Replacing bias and adding it as a second o/p of "
                                     "ConvolutionBiasBackpropFiltersBias";
-                    ngraph::replace_node(delta_user, goe2);
+                    auto goe2_reshape =
```

Comment:
we should check if this is needed... the same way we check above if we have to reshape `bias` into 1D

Reply:
My bad. Done